docs/src/tutorials/supported_operations.md

```diff
@@ -13,14 +13,14 @@ IoT devices often do more than just send data to the cloud. They also do things
 These operations that are supported by [Cumulocity IoT](https://cumulocity.com/api/10.11.0/#section/Device-management-library) and other cloud providers.
 On `thin-edge.io` the support for one such operation can be added using the `thin-edge.io` Supported Operations API.
 
-### thin-edge.io Supported Operations API
+### `thin-edge.io` Supported Operations API
```

Comment:
Overall, **thin-edge.io** is not a command nor a piece of code, rather a product name. I prefer to bold them.

```suggestion
### **thin-edge.io** Supported Operations API
```